NEWS.md

```diff
@@ -26,6 +26,7 @@
 ## Documentation
 
 - Brought the docs on `alpha_sd` up to date with the code change from prior PR #853. By @zsusswein in #862 and reviewed by @jamesmbaazam.
+- All examples now use the natural parameters of distributions rather than the mean and standard deviation to remove warnings and encourage best practice. By @jamesmbaazam in #893 and reviewed by @.
```

Comment:
```suggestion
- All examples now use the natural parameters of distributions rather than the mean and standard deviation and fixed instead of uncertain parameters to remove warnings and encourage best practices. By @jamesmbaazam in #893 and reviewed by @sbfnk.
```